packages/react-core/src/components/Toolbar/Toolbar.tsx

```diff
@@ -50,6 +50,8 @@ export interface ToolbarProps extends React.HTMLProps<HTMLDivElement>, OUIAProps
   ouiaId?: number | string;
   /** Set the value of data-ouia-safe. Only set to true when the component is in a static state, i.e. no animations are occurring. At all other times, this value must be false. */
   ouiaSafe?: boolean;
+  /** Color variant of the toolbar */
+  colorVariant?: 'no-background' | 'primary' | 'secondary';
```

Comment:
Question for react-dev - should we have a `default` which will unset `colorVariant`? Without a `colorVariant` set, the toolbar background can be dynamic depending on how it's used. By default, a toolbar's background is transparent (same as `colorVariant="no-background"`) except when it's sticky, then it's the primary background (same as `colorVariant="primary"`)

Reply:
I would usually lean towards having the "default" case be where a user would leave out the `colorVariant` prop entirely, and only apply the colors if the prop is present. But we've done both ways in different areas, so either works.

Reply:
I did a quick search to see what we do with other color variant props.  It looks like the Drawer has a color variant that has a `default` option.  So I would lean towards adding one here as well for consistency.